packages/shared/src/lib/featureManagement.ts

```diff
@@ -122,8 +128,30 @@ export class Features {
 
   static readonly MyFeedOnboardingVersion = new Features(
     'my_feed_onboarding_version',
-    'control',
-    ['control', 'v1', 'v2', 'v3'],
+    MyFeedOnboardingVersionEnum.Control,
+    [
+      MyFeedOnboardingVersionEnum.Control,
+      MyFeedOnboardingVersionEnum.V1,
+      MyFeedOnboardingVersionEnum.V2,
+      MyFeedOnboardingVersionEnum.V3,
+    ],
+  );
+
+  static readonly OnboardingSteps = new Features(
+    'onboarding_steps',
+    'topics/layout/theme',
```

Comment:
Rather use the enums you defined for this 🙏

Reply:
This is actually the combination of enums, as this will be coming from the flagsmith. But when we expose it to our code, this is converted to `enum[]`

Reply:
This is the default right? That's why I was thinking perhaps we can typecast this one a bit safer?

Reply:
Yes, this is the default, but I'm quite not yet sure of what would be the values for the enum.

Currently, we do the typecasting before we pass it to our context provider as below.
![image](https://user-images.githubusercontent.com/13744167/198282607-7f5497d7-c22f-4843-a367-09917161a106.png)

Reply:
I have now tried to concatenate it this way. WDYT?

![image](https://user-images.githubusercontent.com/13744167/198512021-f0b3c933-846d-473e-a496-4f870fc48bc3.png)